Pinta.Docking/Pinta.Docking.csproj

```diff
@@ -1,17 +1,110 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <Nullable>enable</Nullable>
 
-    <RootNamespace>Pinta.Docking</RootNamespace>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <OutputType>Library</OutputType>
     <AssemblyName>Pinta.Docking</AssemblyName>
+    <RootNamespace>Pinta.Docking</RootNamespace>
+
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ErrorReport>prompt</ErrorReport>
+    <GenerateDocumentation>true</GenerateDocumentation>
+    <NoWarn>1591</NoWarn>
+    <OutputPath>..\bin</OutputPath>
+    <Platform>AnyCPU</Platform>
+    <WarningLevel>4</WarningLevel>
+    <NoStdLib>false</NoStdLib>
+  </PropertyGroup>
+
+  <!-- Debug configuration -->
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <DefineConstants>DEBUG</DefineConstants>
+    <DocumentationFile>..\bin\netcoreapp3.1\Pinta.Docking.xml</DocumentationFile>
+    <Optimize>false</Optimize>
+  </PropertyGroup>
 
-    <Version>1.8.0.0</Version>
-    <Authors>PintaProject</Authors>
-    <Company />
-    <Product />
+  <!-- Release configuration -->
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugSymbols>false</DebugSymbols>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <DocumentationFile>..\bin\netcoreapp3.1\Pinta.Docking.xml</DocumentationFile>
   </PropertyGroup>
+
+  <!-- References -->
+  <ItemGroup>
+    <Reference Condition=" '$(PINTA_MONO_POSIX_2)' != '' " Include="Mono.Posix, Version=2.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756, processorArchitecture=MSIL" />
```

Comment:
This might have been copied over from the master branch, but isn't necessary for GTK3
Could you revert some of this file to match the current gtk3-v2 branch? e.g. adding back the
```
    <Version>1.8.0.0</Version>
    <Authors>PintaProject</Authors>
    <Company />
    <Product />
```